### PR TITLE
docs/variants/msi_z690/: add MSI Z690-A DDR5 to test and HW matrixes

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: sulewskiprzemyslaw <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/variants/msi_z690/faq.md [hunk 1]
```diff
@@ -7,14 +7,13 @@ Reddit.
...
+Not yet. The build is configured for the DDR4 only. But currently we work on
+the DDR5 variant. **Do not try that**.
 
 ## Will this work on my `MSI PRO Z690-A DDR4` ?
 
-Although we do not own that one and cannot give promises without testing, we
-believe that it should work the same as the `MSI PRO Z690-A WIFI DDR4`. This is
-almost the same board, just with no WiFi card.
+Yes, it will work on your platform. Working on MSI PRO Z690-A DDR have
+been confirmed during the test procedures
```
Comment (artur-rs) [inline docs/variants/msi_z690/faq.md:16]: @sulewskiprzemyslaw missing dot

File: docs/variants/novacustom_nv4x/overview.md [hunk 2]
```diff
@@ -40,9 +40,9 @@ in the [release page](releases.md).
...
     Dasharo firmware validation procedure.
-- [Post-installation setup](post_install.md) - describes the post installation
-    steps which should be performed to unlock the full functionality of the
-    product.
+- [Post-installation setup](/common-coreboot-docs/clevo_post_install/) -
+    describes the post installation steps which should be performed to unlock
+    the full functionality of the product.
 - [Updating firmware using fwupd](/common-coreboot-docs/fwupd_usage/) -
     describes the method of updating the Dasharo firmware by using fwupd daemon.
 - [Logo customization](/common-coreboot-docs/custom_logo.md) - describes the
```
Comment (Pokisiekk) [inline docs/variants/novacustom_nv4x/overview.md:48]: These three references don't work after building the site locally.

File: docs/variants/msi_z690/faq.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/msi_z690/faq.md:16]: added

File: docs/variants/novacustom_nv4x/overview.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/novacustom_nv4x/overview.md]: @Pokisiekk now, all links should work properly

**Review by Pokisiekk — APPROVED** [review]

**Pull request merged**